src/quo2/core.cljs

```diff
@@ -176,7 +177,7 @@
 
 ;;;; Colors
 (def color-picker quo2.components.colors.color-picker.view/view)
-(def picker-colors quo2.components.colors.color-picker.view/picker-colors)
+(def color quo2.components.colors.color.view/view)
```

Comment:
perhaps we should add a comment to this export to explain it's use, as it's slight breaking the pattern of quo2 interface. 
cc @ajayesivan

Reply:
Are you talking about `picker-colors`? I removed it/